src/Streamly/Internal/Console/Stdio.hs

```diff
@@ -69,7 +69,7 @@ read = Unfold.lmap (\() -> stdin) Handle.read
 -- > getBytes = Handle.toBytes stdin
 -- > getBytes = Stream.unfold Stdio.read ()
 --
--- /Pre-release/
+-- @since 0.8.0
```

Comment:
Only those APIs that are exposed to users through Streamly.Console.Stdio should be marked as 0.8.0, others should remain `Pre-release`.